benchmark_runner/common/oc/oc.py

```diff
@@ -383,6 +383,15 @@ def get_prom_token(self):
             prom_token = self.run(f"{self.__cli} sa new-token -n openshift-monitoring prometheus-k8s 2>/dev/null")
         return prom_token
 
+    def collect_events(self):
+        """
+        This method collect event logs
+        :return: output_filename
+        """
+        output_filename = os.path.join(self._run_artifacts, f'events.logs')
+        self.run(f"{self.__cli} get events -A > {output_filename} ")
```

Comment:
Quote and trailing space character:

```
self.run(f"'{self.__cli}' get events -A > '{output_filename}'")
```

Reply:
I have added it only to '{output_filename}', for 'oc' it will cause syntax's issue and also we pass this parameter.

Reply:
Have you tested that?

Reply:
I dont see any reason to put 1 value parameter in quotes

Reply:
You right, for {output_filename} I have already added in all the places, but {self.__cli} is 1 parameter and not a file name.

Reply:
It could be, at least in principle.  Suppose self.__cli were `/usr/Here I Am/bin/oc`?

Reply:
The default value comes with [1 parameter](https://github.com/redhat-performance/benchmark-runner/blob/7f1c653d63006f234bc9a76d44af5ead3ff266da/benchmark_runner/main/environment_variables.py#L276-L282) no one should change it.

Reply:
```
 self._environment_variables_dict['cli'] = EnvironmentVariables.get_env('CLI', 'oc')
```

What if someone sets CLI to something that has a space in it?

Reply:
Then why does it accept an environment variable at all?

Reply:
We can change it in case of major change, but its something that under our control

Reply:
So it's possible for the user to change.  There's no harm in programming defensively.

Reply:
(Strictly speaking, this isn't perfect either -- it's possible, although very rare, for a pathname to contain a quote character.  The really safe thing to do is to actually pass the arguments as a list, rather than as a string, but that's a separate issue.)